querymsc.py

```diff
@@ -505,5 +507,124 @@ def query_rain_rate_one_day_1_50(self, session):
                                   History.lon,
                                   History.station_id)
                  )
- 
-        return query
\ No newline at end of file
+  
+        return query
+
+class Gumbel:
+    """This class is meant to do all the fitting
+    of Gumbel distributions to extract design values.
+    
+    Attributes:
+        return_period (int): return period in years
+        min_fit (int): minimum number of years to consider
+            fitting Gumbel.
+    """
+
+    def __init__(self, return_period, min_fit=10):
+
+        self.return_period = return_period
```

Comment:
Following up on @rod-glover's comment that this shouldn't be a class, here is the best evidence. The class only has two instance properties, `return_period` and `min_fit`, each of which are only used one single time. If they're only used once, never updated, you could just as easily (*much more* easily) simply include them as a parameter to the functions that need them.

Reply:
Rod and I just had a chat, and after explaining the code, I think we've uncovered a larger design discussion for the tools. We were hoping to chat with you further about this.